csharp/src/Ice/Properties.cs

```diff
@@ -165,15 +165,17 @@ public string getPropertyWithDefault(string key, string value)
 
     /// <summary>
     /// Get a property as an integer.
-    /// If the property is not set, 0 is returned.
+    /// If the property is not set, 0 is returned. Throws PropertyException if the property
```

Comment:
The summary should not describe what the method throws.

Reply:
Still have to move the exception docs to `exception` tag